### ISSUE TITLE
Add filter by channel in IPC panel

### ISSUE THREAD

**Issue by parro-it** [electron/devtron#63]
"Hide internal button" is useful, but it will be useful if we can filter events by channel name, something similar to the filter feature in console panel.

**Comment by kevinsawicki** [comment]
Yeah, the filter box matches on channel name and data so if you only want to see events for one channel, you can put that in the filter box.

More advanced filtering would be useful though, 👍

**Comment by parro-it** [comment]
@kevinsawicki I didn't notice the actual text filter match also channel name. This is useful, but I think it would be better to make it filter only on messages content, and add a select box for channel names.

**Comment by parro-it** [comment]
Awesome work by the way!
